src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id: number;
+  userId: number;
+  title: string;
+  completed: boolean;
+}
```

Comment:
I suggest also creating a type Todoid and use it to type todo's id everywhere for better readability

```suggestion
export type TodoId = number;

export interface Todo {
  id: TodoId;
  userId: number;
  title: string;
  completed: boolean;
}
```